docs/ja/README.md

```diff
@@ -0,0 +1,99 @@
+---
+home: 'true'
+heroImage: "/logo.svg"
+actionText: Get Started →
+actionLink: "/guide/"
+footer: ISC Licensed | Copyright © 2019-present Starship Contributors
+---
+
+<div class="features">
+  <div class="feature">
+    <h2>互換性第一</h2>
```

Comment:
@unhappychoice I commented in [GitLocalize – Continuous Localization for GitHub Projects](https://gitlocalize.com/repo/3018/ja/review/10411).

What do you think?

Reply:
Thanks I replied for that.
As I mentioned there, I think we should translate words as much as possible to help people who don't understand English, instead of keeping the words as English for styling or meaning accuracy.